app-backend/migrations/src/main/scala/migrations/29.scala

```diff
@@ -0,0 +1 @@
+../../../../src_migrations/main/scala/29.scala
```

Comment:
Need to watch out, #756 is going to have a migration conflict. I think the rebase conflict will stop the merge, but we should be careful.